tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -894,6 +908,18 @@ Status TrtNodeValidator::ConvertToTensorOrWeights(
     std::vector<TRT_TensorOrWeights> inputs;
     return ConvertConstToWeights(node_def, inputs, tensor_or_weights);
   }
+  if (node_def.op() == "ReadVariableOp") {
+    // Similar treatment to Const and VariableV2, but we provide a fake
+    // resource input to the converter.
+    Status status;
+
+    std::vector<TRT_TensorOrWeights> inputs;
+    ResourceHandle fake_resource;
+    inputs.emplace_back(fake_resource);
```

Comment:
Can we please try replacing with `const std::vector<TRT_TensorOrWeights> inputs{ResourceHandle()};` (and marking const if possible)?

For an example see: https://godbolt.org/z/389YaKTGf

Reply:
Yes, although the correct syntax is: `const std::vector<TRT_TensorOrWeights> inputs{TRT_TensorOrWeights(ResourceHandle())};`